src/pint/pintk/plk.py

```diff
@@ -1068,11 +1068,11 @@ def plotResiduals(self, keepAxes=False):
                     else self.psr.postfit_resids
                 )
                 if self.y_unit == u.us:
-                    f0 = r.get_PSR_freq().to(u.MHz).value
+                    f0 = r.model.F0.quantity.to(u.MHz).value
```

Comment:
Should this be `r.model.get_spin_freq()` with some appropriate options? At a minimum so we can easily identify this as a place we do that conversion?

Reply:
Maybe?  But `get_spin_freq` requires times to be specified.  What time are we going to use for plotting?  (and that is all this is used for:  the 2nd axis of the residual plots)